server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/Dequeuer.java

```diff
@@ -90,31 +90,29 @@ public void done(boolean success) {
     }
 
     Flux<? extends MailQueue.MailQueueItem> deQueue() {
-        return flux.concatMap(this::loadItem)
-            .concatMap(this::filterIfDeleted);
+        return flux.flatMapSequential(this::loadItem)
```

Comment:
what about:
```java
return flux.flatMapSequential(response -> loadItem(response).flatMap(this::itemWithIsPresent))
```
to avoid doing several times the sequential reordering?

Reply:
`flux` is obscure, can you rename it?